src/Data/JSDate.purs

```diff
@@ -150,8 +150,11 @@ foreign import dateMethod :: forall a. Fn2 String JSDate a
 
 -- | Attempts to parse a date from a string. The behaviour of this function is
 -- | implementation specific until ES5, so may not always have the same
--- | behaviour for a given string. The RFC2822 and ISO8601 date string formats
```

Comment:
`behaviour` &rarr; `behavior`

Does PureScript have any language-level conventions on spelling?

Reply:
I'm not aware of any conventions; both British / American English variants appear in various places. I don't mind which one is used here, but if we switch this `behaviour` -> `behavior` then we should adjust the one two lines up as well.

Reply:
Good catch!